lib/moon/design/snackbar/trigger.ex

```diff
@@ -0,0 +1,18 @@
+defmodule Moon.Design.Snackbar.Trigger do
+  @moduledoc false
+
+  use Moon.StatelessComponent, slot: "trigger"
+
+  # slot(default, required: true)
```

Comment:
Leaving in commented code is generally considered a code smell. And this seems like an extreme case, the whole body commented out?